grafana/provisioning/dashboards/metrics.json

```diff
@@ -907,56 +915,571 @@
+                    "type": "timeseries"
+                }
+            ],
+            "title": "Node stats",
+            "type": "row"
+        }
+    ],
+    "refresh": "",
+    "schemaVersion": 38,
+    "tags": [],
+    "templating": {
+        "list": [
+            {
+                "current": {
+                    "selected": false,
+                    "text": "Prometheus",
+                    "value": "PBFA97CFB590B2093"
+                },
+                "hide": 0,
+                "includeAll": false,
+                "multi": false,
+                "name": "ds",
+                "options": [],
+                "query": "prometheus",
+                "queryValue": "",
+                "refresh": 1,
+                "regex": "",
+                "skipUrlSync": false,
+                "type": "datasource"
+            },
+            {
+                "current": {
+                    "selected": true,
+                    "text": [
+                        "All"
+                    ],
+                    "value": [
+                        "$__all"
+                    ]
+                },
+                "datasource": {
+                    "type": "prometheus",
+                    "uid": "PBFA97CFB590B2093"
+                },
+                "definition": "label_values(jellyfish_rooms,instance)",
+                "description": "Allows you limit graphs only to selected instances",
+                "hide": 0,
+                "includeAll": true,
+                "label": "Instance",
+                "multi": true,
+                "name": "instance",
+                "options": [],
+                "query": {
+                    "query": "label_values(jellyfish_rooms,instance)",
+                    "refId": "PrometheusVariableQueryEditor-VariableQuery"
+                },
+                "refresh": 1,
+                "regex": "",
+                "skipUrlSync": false,
+                "sort": 0,
+                "type": "query"
+            },
+            {
+                "current": {
+                    "selected": true,
+                    "text": [
+                        "All"
+                    ],
+                    "value": [
+                        "$__all"
+                    ]
+                },
+                "datasource": {
+                    "type": "prometheus",
+                    "uid": "PBFA97CFB590B2093"
+                },
+                "definition": "label_values(node_uname_info,instance)",
+                "description": "Allows you limit graphs only to selected nodes",
+                "hide": 0,
+                "includeAll": true,
+                "label": "Node",
+                "multi": true,
+                "name": "node",
+                "options": [],
+                "query": {
+                    "qryType": 1,
+                    "query": "label_values(node_uname_info,instance)",
```

Comment:
I'm not sure if instance regards actual node or instance of node exporter (IMO the latter). Better pick (IMHO) would be `nodename` or `domainname` (node name is the value of `hostname` typed in the container, whereas I don't have domainname).

Reply:
I chose `instance` because it is easier to understand on which node this is happening and compare it with stats from our containers.

Reply:
I proposed `nodename` since that's what descriptive name of the node is in my company's metrics. My only fear here was if `instance` wasn't pointing to node exporter. Although node exporters probably work one by node and those should be easy to find like that.